packages/utilities/web-vitals/README.md

```diff
@@ -0,0 +1,75 @@
+# web-vitals - [![Known Vulnerabilities](https://snyk.io/test/github/bbc/psammead/badge.svg?targetFile=packages%2Fcomponents%2Fweb-vitals%2Fpackage.json)](https://snyk.io/test/github/bbc/psammead?targetFile=packages%2Fcomponents%2Fweb-vitals%2Fpackage.json) [![Dependency Status](https://david-dm.org/bbc/psammead.svg?path=packages/components/web-vitals)](https://david-dm.org/bbc/psammead?path=packages/components/web-vitals) [![peerDependencies Status](https://david-dm.org/bbc/psammead/peer-status.svg?path=packages/components/web-vitals)](https://david-dm.org/bbc/psammead?path=packages/components/web-vitals&type=peer) [![Storybook](https://raw.githubusercontent.com/storybooks/brand/master/badge/badge-storybook.svg?sanitize=true)](https://bbc.github.io/psammead/?path=/story/web-vitals--containing-image) [![GitHub license](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/bbc/psammead/blob/latest/LICENSE) [![npm version](https://img.shields.io/npm/v/@bbc/web-vitals.svg)](https://www.npmjs.com/package/@bbc/web-vitals) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/bbc/psammead/blob/latest/CONTRIBUTING.md)
+
+## Description
+
+The `WebVitals` utility provides a configurable hook which will collect device capability metrics (effective network type, device CPU count, and device memory) and Web Vitals metrics (Cumulative Layout Shift, Time To First Byte, etc) for each page view, and then send those metrics to a reporting endpoint when the user leaves the page.
```

Comment:
Is it worth listing out all of the web-vitals metrics here? And perhaps a link to the web-vitals docs for information (you may be able to copy & paste directly from the issue! https://github.com/bbc/psammead/issues/3778